### PR TITLE
Changelog update for 2.2.0next

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+## v2.x.x [202x-xx-xx]
+
+### Added
+
+### Changed
+
+- [FOGL-7475] System Logs page search improvements to search data from backend through API [#270](https://github.com/fledge-iot/fledge-gui/pull/270)
```
Comment (praveen-garg) [inline docs/changelog.md:7]: this should be in the last release

File: docs/changelog.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+
+- [FOGL-7948] Configuration page single tree structure for Categories. Removed the dropdown selector for root categories/parent groups [#330](https://github.com/fledge-iot/fledge-gui/pull/330)
+
+### Fixed
+
+- [FOGL-8199]: Same datapoints/attribute for different asset not displaying correctly in the graph [#347](https://github.com/fledge-iot/fledge-gui/pull/347)
```
Comment (praveen-garg) [inline docs/changelog.md:13]: Already mentioned in 2.2.1

File: docs/changelog.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+
+- [FOGL-7368]: Added left & right slider indicator when configuration group tabs are not visible [#250](https://github.com/fledge-iot/fledge-gui/pull/250)
```
Comment (praveen-garg) [inline docs/changelog.md:15]: this should be in the last release

File: docs/changelog.md (hunk 1 above)
Comment (praveen-garg) [reply docs/changelog.md:7]: We also have added link to services and logs stuff

**Review by monika-Dianomic — APPROVED** [review]

**Review by hemantgarg923 — APPROVED** [review]

**Pull request merged**